packages/manager/apps/public-cloud/src/sidebar/sidebar.less

```diff
@@ -0,0 +1,143 @@
+@background-color: rgb(244,248,251);
+@text-color: rgb(17,63,109);
+
+cloud-sidebar { /* stylelint-disable-line */
+  background-color: @background-color;
+  color: @text-color;
+  width: 100%;
+  height: 100%;
+
+  oui-dropdown-content { /* stylelint-disable-line */
+    color: @text-color;
+    width: 90%;
+
+    ul {
+      list-style-type: none;
+      padding: 0;
+      margin: 0;
+      max-height: 50vh;
```

Comment:
Is this menu can be reach in mobile?

If yes then I suggest to increase this value if we are in `orientation: landscape` by using media query.

Do you feel agree with that?

Reply:
Agreed.

Currently, mobile is not working correctly. Still in work in progress :)

-> Should be managed by the "new" navbar, used by all managers :)